docs/Interoperator-metrics.md

```diff
@@ -23,6 +24,14 @@ An example for adding custom metric can be found in pull request [#983](https://
 * In `SetupWithManager` section of the controller, register the `collector` with the prometheus [Registry](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/metrics) using the `metrics.Registry.MustRegister()` function. `MustRegister()` accepts variable number of arguments and all the `collectors` can be registered together. Import the `"sigs.k8s.io/controller-runtime/pkg/metrics"` package to access the global `Registry`.
 * Anywhere in the reconcile loop, update the metric.
 
+### Sample "interoperator_service_bindings_state" promQL queries and their corresponding outputs:
+Query | Output | Comment
+--- | ---- | --- |
+interoperator_service_bindings_state | interoperator_service_bindings_state{app_kubernetes_io_managed_by="Helm", binding_id="0abc2107-de86-408d-8617-800935b84028", control_plane="interoperator-multiclusterdeployer-controller-manager", instance="100.96.0.48:8443", instance_id="80ca2362b-6561-4673-ad24-111d7ac86cfd", job="kubernetes-service-endpoints", kubernetes_name="interoperator-multiclusterdeployer-metrics-service", kubernetes_namespace="interoperator", kubernetes_node="shoot--interop--pooja-dev-worker-j5mor-z1-7d74d-82vg7"} 1 <br><br> interoperator_service_bindings_state{app_kubernetes_io_managed_by="Helm", binding_id="0abc2107-de86-408d-8617-800935b84038", control_plane="interoperator-multiclusterdeployer-controller-manager", instance="100.96.0.48:8443", instance_id="80ca2362b-6561-4673-ad24-111d7ac86cfd", job="kubernetes-service-endpoints", kubernetes_name="interoperator-multiclusterdeployer-metrics-service", kubernetes_namespace="interoperator", kubernetes_node="shoot--interop--pooja-dev-worker-j5mor-z1-7d74d-82vg7"} 1 <br><br> interoperator_service_bindings_state{app_kubernetes_io_managed_by="Helm", binding_id="0ceb2107-de86-408d-8617-800935b84108", control_plane="interoperator-multiclusterdeployer-controller-manager", instance="100.96.0.48:8443", instance_id="65ca2362b-6561-4673-ad24-111d7ac86cfd", job="kubernetes-service-endpoints", kubernetes_name="interoperator-multiclusterdeployer-metrics-service", kubernetes_namespace="interoperator", kubernetes_node="shoot--interop--pooja-dev-worker-j5mor-z1-7d74d-82vg7"} 0 <br><br> interoperator_service_bindings_state{app_kubernetes_io_managed_by="Helm", binding_id="1abc2107-de86-408d-8617-800935b84038", control_plane="interoperator-multiclusterdeployer-controller-manager", instance="100.96.0.48:8443", instance_id="81ca2362b-6561-4673-ad24-111d7ac86cfd", job="kubernetes-service-endpoints", kubernetes_name="interoperator-multiclusterdeployer-metrics-service", kubernetes_namespace="interoperator", kubernetes_node="shoot--interop--pooja-dev-worker-j5mor-z1-7d74d-82vg7"} 0 | List all the bindings present in the cluster
```

Comment:
The output of the `interoperator_service_bindings_state` is long. I feel we need to show only the relevant parts of the output in the example. May be like below,
```
interoperator_service_bindings_state{binding_id="0abc2107-de86-408d-8617-800935b84028", instance_id="80ca2362b-6561-4673-ad24-111d7ac86cfd"} 1
```
This will improve the readability.

Reply:
Thanks.. I too felt the same but wasn't sure if truncated output was allowed. Updated the doc.